js/components/closed-caption-multi-audio-menu/tab.js

```diff
@@ -19,24 +22,36 @@ var Tab = React.createClass({
           speed={1}
           horizontal={false}
         >
-          <ul className="oo-cc-ma-menu__list">
+          <ul
+            className="oo-cc-ma-menu__list"
+            role={CONSTANTS.ARIA_ROLES.MENU}
+          >
             {this.props.itemsList.map(function(el, index) {
               return (
                 <li
                   key={index}
```

Comment:
Is it possible to use `el.id` instead of `index` here? Using indexes as keys is not recommended: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
\+

Reply:
Very interesting! Thank you!